static/templates/actors/party/sheet.hbs

```diff
@@ -56,6 +56,11 @@
 
             <section class="inventory">
                 {{> "systems/pf2e/templates/actors/partials/coinage.hbs" owner=@root.owner}}
+                <header class="inventory-header">
+                    <div class="search">
+                        <input type="search" spellcheck="false" placeholder="{{localize "PF2E.Actor.Stash.Search"}}" />
+                    </div>
+                </header>
```

Comment:
if it will be part of every inventory, it might be wise to move this code to inventory.hbs, both for this and the one in character itself.